.github/workflows/ci.yml

```
@@ -0,0 +1,79 @@
+name: Static Analysis
```

```suggestion
name: CI Checks
```

Or something like that. Open to other ideas, but this is no longer just static analysis.